.eslintrc.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "tsconfig.json",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint",
+        "prettier",
+        "implicit-dependencies"
+    ],
+    "extends": [
+        "plugin:prettier/recommended"
+    ],
+    "rules": {
+        "@typescript-eslint/class-name-casing": "error",
+        "@typescript-eslint/indent": "off",
+        "@typescript-eslint/no-explicit-any": "error",
+        "@typescript-eslint/no-inferrable-types": "error",
+        "@typescript-eslint/no-require-imports": "error",
+        "@typescript-eslint/no-this-alias": "error",
+        "@typescript-eslint/no-var-requires": "error",
+        "@typescript-eslint/type-annotation-spacing": "off",
+        "arrow-parens": [
+            "off",
+            "as-needed"
+        ],
+        "camelcase": ["error", {
+           "properties": "never",
+            "allow": ["read_only", "env_file", "depends_on", "working_dir"]
+        }],
+        "constructor-super": "error",
+        "curly": "error",
+        "dot-notation": "error",
+        "eol-last": "off",
```

Comment:
Is there a reason these are disabled in the configuration instead of using `eslint-config-prettier`? It looks like you'd only need to add the plugin to your extends array.

Reply:
Ah, I see what you're saying. No, no special reason other than "that's what npx tslint-to-eslint-config gave me." I'm good with using eslint-config-prettier and shortening this list a bit.

Reply:
Let's do that (and for the forumone-cli PR as well).